### DISCUSSION TITLE
How to remove range selection?

### DISCUSSION THREAD

**Discussion by Abdillah** [helix-editor/helix#7378] [Q&A] [answered]
Is there a way to convert selection into single normal cursor?

For example, after yanking I preferred to remove the selection. And then after switching from select mode to normal mode I might try to remove the selection.

**Comment by kirawi** [comment, marked as answer]
Press `;` to collapse your selection

**Reply by Abdillah** [reply]
Right! I got it skipped due to unfamiliarity with collapse.
Thanks.
